README.md

```diff
@@ -109,6 +109,27 @@ Environment config:
 - ``AUTH``: authentication to edit the ``public-suffix-list`` collection
 
 
+### sync_megaphone
+
+Send the current version of Remote Settings data to the Push server.
+
+Does nothing if versions are in sync.
+
+Environment config:
+
+- ``SERVER``: server URL (default: ``http://localhost:8888/v1``)
+- ``MEGAPHONE_HOST``: Push service host (eg.: ``push.services.mozilla.com``)
```

Comment:
The Megaphone domain name is megaphone.services.mozilla.com; what you have is the push service. I guess this is why you are using a websocket below?

Reply:
Oh thanks! That was the source of the confusion! 

I simplified everything using Megaphone :100: